anabeesh/src/main/java/com/cs18/anabeesh/ui/login/LoginActivity.java

```diff
@@ -44,9 +55,63 @@ public void onCreate(@Nullable Bundle savedInstanceState) {
 
     @OnClick(R.id.btn_login)
     void login() {
+
+        userEmail = emailEditText.getText().toString();
+        userPassword = passwordEditText.getText().toString();
+
+        if (userEmail.matches("")) {
+            // TODO : Handling empty e-mail field .
+        }
+        else if (userPassword.matches("")) {
+            // TODO : Handling empty password field .
+        }
+        else if (userPassword.matches("") && userEmail.matches("")) {
+            // TODO : Handling empty e-mail and password field .
+        }
+        else {
+            logInUserRequest = new LogInUserRequest();
+            logInUserRequest.setUserEmail(userEmail);
+            logInUserRequest.setUserPassword(userPassword);
```

Comment:
Balash set, ya emma builder ya emma constructor

Reply:
done . ana 3mltha @Field we keda afdl